ios-target-app/Onboarding/Views/SignInViewController.swift

```diff
@@ -155,8 +213,10 @@ class SignInViewController: UIViewController {
     switch sender {
     case emailField:
       viewModel.fieldChanged(field: .email, value: newValue)
+      viewModel.setSignInButtonStatus()
```

Comment:
this is a sign that the approach is not very reactive. Once the view model receives fieldChanged, it should trigger the corresponding signals. PLs see my other comment above about signals